readthedocs/settings/base.py

```diff
@@ -55,9 +55,14 @@ class CommunityBaseSettings(Settings):
     DEFAULT_FROM_EMAIL = 'no-reply@readthedocs.org'
     SERVER_EMAIL = DEFAULT_FROM_EMAIL
 
-    # Cookies
+    # Sessions
     SESSION_COOKIE_DOMAIN = 'readthedocs.org'
     SESSION_COOKIE_HTTPONLY = True
+    SESSION_COOKIE_AGE = 30 * 24 * 60 * 60  # 30 days
+    SESSION_ENGINE = 'django.contrib.sessions.backends.signed_cookies'
```

Comment:
I'm thinking of actually taking this change out and making this a setting that is set in the production settings rather than in the base settings.

Reply:
There are advantages to signed cookies but they aren't huge and they are what's holding this up due to the complications with the corporate site.

Reply:
I removed the `SESSION_ENGINE` setting here. If we decide to change the production `SESSION_ENGINE`, we'll do that in the production settings file. This will ensure that anyone taking our code and using it don't have a `SESSION_ENGINE` change unless they want one.

Reply:
Makes sense. 👍